patches/chrome-browser-resources-tools-optimize_webui.gni.patch

```diff
@@ -10,3 +10,11 @@ index 5bcdb42a7d8f1443fefa4c81eb7e9757bc45eaa0..3c6e34fc42775ef23e50e7c7b8828350
  
      # Note that we have to manually pass the sources to our script if the
      # script needs them as inputs.
+@@ -78,6 +79,7 @@ template("optimize_webui") {
+       "chrome://resources/polymer/v3_0/|$polymer_path",
+       "chrome://resources/|$resources_path",
+     ]
++    brave_resources_path = rebase_path("$root_gen_dir/brave/ui/webui/resources/", root_build_dir) external_paths += [ "chrome://brave-resources/|$brave_resources_path" ]
```

Comment:
this can be put on the same line with the existing patch. but `brave_resources_path` should be defined elsewhere in a gni file and it should be `external_paths += brave_webui_external_paths`

Reply:
Addressed by 3f9e73e241993ded09525942c15f06a64db64255